### PR TITLE
fix: boot query promise race

### PR DESCRIPTION
**Pull request by sshanzel**

### CODE REVIEW

File: src/routes/boot.js [hunk 1]
```diff
@@ -156,9 +156,9 @@ const getFeaturesForUser = async (ctx) => {
 };
 
 router.get('/', async (ctx) => {
-  const [flags, base, alerts] = await Promise.all([
+  const base = await bootSharedLogic(ctx);
```
Comment (idoshamun) [inline src/routes/boot.js:159]: There is no reason to wait for this function to execute.
We only need to extract the refresh token part and the rest we can do in parallel.

**Review by idoshamun — CHANGES_REQUESTED** [review]

File: src/routes/boot.js (hunk 1 above)
Comment (sshanzel) [reply src/routes/boot.js:159]: Good point.

**Comment by sshanzel** [comment]
The tests turned green! 🟢  this should be now ready for review. Thank you! @idoshamun @rebelchris

**Comment by sshanzel** [comment]
@idoshamun thank you for the awesome session! The checking of the token should now be as minimal as possible

**Review by idoshamun — APPROVED** [review]
Looks great!!! Let's merge

**Pull request merged**